examples/window/window_settings.rs

```diff
@@ -12,13 +13,47 @@ fn main() {
             present_mode: PresentMode::AutoVsync,
             ..default()
         })
+        .insert_resource(VSync(true))
```

Comment:
I think we could avoid this (potentially confusing) extra resource resource by doing something like
```rust
let window = windows.get_primary_mut().unwrap();

window.set_present_mode(if matches!(window.present_mode(), PresentMode::AutoVsync) {
    PresentMode::AutoNoVsync
} else {
    PresentMode::AutoVsync
});
```
down below

Reply:
Cleaned, thanks